hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/table/HoodieFlinkTable.java

```diff
@@ -53,6 +53,27 @@ protected HoodieFlinkTable(HoodieWriteConfig config, HoodieEngineContext context
     super(config, context, metaClient);
   }
 
+  @Override
```

Comment:
This just been moved; no changes (there were other changes, but these were reverted back)